brainframe_qt/constants/oauth.py

```diff
@@ -0,0 +1,10 @@
+COGNITO_DOMAIN = "auth-aotuai-staging.auth.us-east-1.amazoncognito.com"
```

Comment:
@velovix What should this URL be?

Reply:
The production URL is `auth-aotuai-prod.auth.us-east-1.amazoncognito.com`.

Reply:
Just make sure to change these values to production and you'll be good to go!